queries.sql

```diff
@@ -8,3 +8,27 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered IS true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
+UPDATE animal SET specie = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT MAX(escape_attempts) FROM animals;
+SELECT MIN(weight_kg), MAX(weight_kg) FROM animals;
+SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
```

Comment:
- [ ] It appears that the following two are missing from your queries. Kindly include them.

![image](https://user-images.githubusercontent.com/86318284/196690832-fe22b2f1-c252-459d-a377-eb57753443ad.png)

- [ ] It appears the following query is missing kindly add it

![image](https://user-images.githubusercontent.com/86318284/196691614-2c187382-fc96-4d5d-bcc4-ebd92d3cbba7.png)